btcjson/walletsvrcmds.go

```diff
@@ -721,4 +925,5 @@ func init() {
 	MustRegisterCmd("walletlock", (*WalletLockCmd)(nil), flags)
 	MustRegisterCmd("walletpassphrase", (*WalletPassphraseCmd)(nil), flags)
 	MustRegisterCmd("walletpassphrasechange", (*WalletPassphraseChangeCmd)(nil), flags)
+	MustRegisterCmd("importmulti", (*ImportMultiCmd)(nil), flags)
```

Comment:
Other `MustRegisterCmd` lines in this function are in alphabetical order. This `MustRegisterCmd("importmulti"` line should be before the `MustRegisterCmd("importprivkey"` line

Reply:
Indeed. I didn't pay attention to the order.